### DISCUSSION TITLE
Unnecessary scrollbar in keyboard shortcuts modal

### DISCUSSION THREAD

**Discussion by Santeenee** [community/community#69050] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

An unwanted scrollbar on the right appears as I open the modal for keyboard shortcuts in whatever repository (like [GitHub docs network page](https://github.com/github/docs/network)). This seems to happen in chromium browsers.

OS: Windows 10 22H2
Browser: Chrome Version 117.0.5938.92 (Official Build) (64-bit)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by joaorosera** [comment]
Response #2

**Reply by Santeenee** [reply]
I'm curious, what #2 has to do with this discussion?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Santeenee ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
